Can't compile lib

Throws this:
```
error[E0599]: no function or associated item named `new` found for struct `SGD` in the current scope
   --> C:\Users\zxcqirara\.cargo\registry\src\index.crates.io-6f17d22bba15001f\rustpotter-3.0.1\src\wakewords\nn\wakeword_model_train.rs:196:31
    |
196 |     let sgd = candle_nn::SGD::new(var_map.all_vars(), args.learning_rate);
    |                               ^^^ function or associated item not found in `SGD`
    |
    = help: items from traits can only be used if the trait is in scope
help: the following trait is implemented but not in scope; perhaps add a `use` for it:
    |
1   + use candle_nn::Optimizer;
    |
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -14,8 +14,8 @@ exclude = [
 ]
 
 [dependencies]
-candle-core = "0.2.0"
-candle-nn = "0.2.0"
+candle-core = "0.2.1"
+candle-nn = "0.2.1"
 ciborium = "0.2.0"
 hound = "3.4.0"
 rubato = { version = "0.14.1", default-features = false }

--- a/src/wakewords/nn/wakeword_model_train.rs
+++ b/src/wakewords/nn/wakeword_model_train.rs
@@ -8,6 +8,7 @@ use crate::{
     ModelType, WakewordModel,
 };
 use candle_core::{DType, Device, Tensor, D};
+use candle_nn::Optimizer;
 use candle_nn::{loss, ops, VarMap};
 use std::{
     collections::HashMap,
@@ -193,7 +194,7 @@ fn training_loop<M: ModelImpl + 'static>(
         m.labels,
         wakeword.as_ref(),
     )?;
-    let sgd = candle_nn::SGD::new(var_map.all_vars(), args.learning_rate);
+    let mut sgd = candle_nn::SGD::new(var_map.all_vars(), args.learning_rate).unwrap();
     let test_features = m.test_features.to_device(&dev)?;
     let test_labels = m.test_labels.to_dtype(DType::U32)?.to_device(&dev)?;
     if from_wakeword {

